src/coreComponents/mesh/generators/CellBlockUtilities.cpp

```diff
@@ -192,9 +192,9 @@ void getFaceNodes( ElementType const & elementType,
         {
           nodeIndices.resize( 4 );
           nodeIndices[0] = elementToNodes[iElement][0];
-          nodeIndices[1] = elementToNodes[iElement][1];
-          nodeIndices[2] = elementToNodes[iElement][2];
-          nodeIndices[3] = elementToNodes[iElement][3];
+          nodeIndices[1] = elementToNodes[iElement][2];
+          nodeIndices[2] = elementToNodes[iElement][3];
+          nodeIndices[3] = elementToNodes[iElement][1];
```

Comment:
@klevzoff Be aware of this patch since your PR modifies this code too!

Reply:
I decided to jump in first, so now it's a conflict for @castelletto1 to resolve :smile: 
But the change is simple, I just split this long function into sub-functions, one per element shape.